pkg/model/dev.go

```diff
@@ -809,6 +810,21 @@ func (dev *Dev) Validate() error {
 	return nil
 }
 
+// Prepare calls other methods required to have the dev ready to use
+func (dev *Dev) Prepare(manifestPath string) error {
```

Comment:
Given that this is just expanding relative paths. Should we name it `PreparePaths`?